packages/xo-web/src/common/xoa-updater.js

```
@@ -381,6 +381,14 @@ class XoaUpdater extends EventEmitter {
     }
   }
 
+  async getReleaseChannels() {
+    try {
+      return await this._call('getReleaseChannels')
```

To simplify this implementation, i suggest:
```js
return this._call('getReleaseChannels').catch(error => ({}))
```

---

Using `promise-toolbox/ignoreErrors` is better because it does not swallow programmer errors.

---

I agree :+1:

---

All `xoa-updater` methods use `async/await`, I suggest to keep homogeneity

---

@badrAZ what is your last word ?

---

We only use `async/await` if it's need or if it's more understandable. In this case, IMHO, it will increase code complexity.

---

I'm open to discussion.

---

I don't think it's a big deal either way, let's follow the senior dev (@badr) advice on this :slightly_smiling_face:

---

I fixed this as senio dev @badrAZ `wanted`